File: 백준/Bronze/15025. Judging Moose/Judging Moose.c
#include <stdio.h>

int main()
{
    int l, r;

    scanf("%d %d", &l, &r);

    if (l == r && l != 0 && r != 0)
        printf("Even %d", l * 2);
    else if (!l && !r)
        printf("Not a moose");
    else
        printf("Odd %d", l > r ? l * 2 : r * 2);

    return 0;
}

File: 백준/Bronze/15025. Judging Moose/Judging Moose.py
l, r = map(int, input().split())
if l == r and l != 0 and r != 0:
    print("Even", l * 2)
elif l == 0 and r == 0:
    print("Not a moose")
else:
    print("Odd", l * 2 if l > r else r * 2)